src/components/common/NetworkSelector/index.tsx

```diff
@@ -112,6 +114,11 @@ const NetworkSelector = (props: { onChainSelect?: () => void }): ReactElement =>
           '& .MuiPaper-root': {
             overflow: 'auto',
           },
+          ...(theme.palette.mode == 'dark' && {
```

Comment:
We have a hook for this called `useDarkMode`. You can use it like this in the component: `const isDarkMode = useDarkMode()`